conf/mesos-site.xml

```diff
@@ -63,11 +63,21 @@
   </property>
 
   <property>
+    <name>mesos.hdfs.preshared-configuration</name>
```

Comment:
I'm not a fan of this property name, since you're also using it to trigger symlinks and update the path. How about "preloaded-hadoop" or "preinstalled-hadoop", since you wouldn't need to update symlinks/path or reload hadoop configs if they're already preloaded/preinstalled?